routes/[version]/collections/_helpers.js

```diff
@@ -32,12 +32,6 @@ async function saveCollection(userId, data) {
     }
 
     if (users) {
```

Comment:
@Valexr why collection visibility guard was deleted?

Reply:
Because `users` can be only in `shared` collections.

Reply:
Yes, but what if this function was called for a collection with another `visibility`? I think, the guard is needed.